### PR TITLE
Implement Follow , Unfollow functionalites

### LINKED ISSUE #130: Implement Follow Unfollow feature for organization
Implement Follow Unfollow feature for organization

### PR DESCRIPTION
**Pull request by rijusougata13**
I have implemented follow and unfollow functionalities

## Related Issue

Fixed #130 

## Motivation and Context

User can now follow or unfollow a specific organization

## How Has This Been Tested?

with npm (no error)

## Screenshots or GIF (In case of UI changes):
simplescreenrecorder-2021-06-22_15 30 22

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/components/Organization/ViewOrganization/index.js [hunk 1]
```diff
@@ -15,33 +15,104 @@ import { clearOrgData, getOrgData } from "../../../store/actions";
...
+        orgFollowed: arr2,
+      });
+    } else {
+      db.collection("cl_org_general")
+        .doc(profileData.uid)
+        .update({
+          orgFollowed: [handle],
+        });
+    }
+  };
```
Comment (Niweera) [inline src/components/Organization/ViewOrganization/index.js:77]: These database actions are not related to the JSX view, right? Please separate the actions and views into two components and import the actions in the view file. ([Separation of concern](https://nalexn.github.io/separation-of-concerns/)). You can move these actions to `/src/store/actions` directory.

Comment (rijusougata13) [reply src/components/Organization/ViewOrganization/index.js:77]: okay moving those functions  into store/actions directory

Comment (rijusougata13) [reply src/components/Organization/ViewOrganization/index.js]: moved all the functions which are not related to JSX to ```store/actions/orgActions.js```. @Niweera

**Review by Niweera — APPROVED** [review]

**Comment by thisuraseniya** [comment]
Please resolve the merge conflicts and commit again

**Comment by rijusougata13** [comment]
> Please resolve the merge conflicts and commit again

Fixed all the conflicts .

**Review by thisuraseniya — APPROVED** [review]
